test/azul_test_case.py

```diff
@@ -52,6 +56,10 @@ class AzulTestCase(TestCase):
     get_credentials_boto3 = None
     _saved_boto3_default_session = None
 
+    # Region corresponds to south asia (Mumbai). Must be a real region because
+    # SQS will verify this.
```

Comment:
While I believe that something may be checking this value, I don't think it can be SQS since that's mocked out by moto. Do you mean moto?

Reply:
True, but moto defers to boto when retrieving the list of valid regions.